<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_client</class>
 <widget class="QWidget" name="register_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>467</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Lohit Bengali</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cliente</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Correo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Apellido</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Cedula</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Telefono</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Estado</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Ciudad</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Sexo</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Confirmar</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Comida
 Favorita</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_correo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_nombre">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_cedula">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_apellido">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_telefono">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_estado">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_ciudad">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_sexo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Masculino</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Femenino</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_password">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_confirm">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QComboBox" name="comboBox_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Comida Rapida</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Italiano</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Arabe</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Francesa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Española</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>China</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Gourmet</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Grill</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Japonesa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Venezolana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Latina</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mexicana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Postres</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Comida Rapida</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Italiano</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Arabe</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Francesa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Española</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>China</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Gourmet</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Grill</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Japonesa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Venezolana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Latina</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mexicana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Postres</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Comida Rapida</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Italiano</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Arabe</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Francesa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Española</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>China</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Gourmet</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Grill</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Japonesa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Venezolana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Latina</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mexicana</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Postres</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Registrar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Salir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
